teuthology/suite/merge.py

```diff
@@ -115,6 +115,7 @@ def config_merge(configs, suite_name=None, **kwargs):
     the entire job (config) from the list.
     """
     seed = kwargs.setdefault('seed', 1)
+    base_config = kwargs.setdefault('base_config', {})
```

Comment:
```suggestion
    base_config = kwargs.setdefault('base_config', JobConfig())
```
And add `from teuthology.config import JobConfig` up top

Reply:
thanks, done